userguide/virtualmachines.rst

```diff
@@ -61,7 +61,68 @@ and |ui-add| to open the wizard shown in
 .. figure:: images/virtual-machines-add-wizard-type.png
 
    Add VM
-
+   
+   
+Create a virtual machine by following these seven steps of the creation
+wizard:
+
+#. **Select VM wizard type**
+
+   Select :literal:`Virtual Macine (VM)` or
+   :literal:`Docker Host`.
+
+#. **Operating System**
+
+   Select the guest operating system type from the drop down menu and
+   enter an alphanumeric name for the VM. It is recommended to use UEFI
+   as the boot method for newer operating systems. The settings
+   :guilabel:`Start on Boot` and :guilabel:`Enable VNC` are set by default.
+   The setting :guilabel:`Start on Boot` tells the VM to start when the
+   host system is booted. The setting :guilabel:`Enable VNC` allows a
+   virtual connection to the VM.
+   
+#. **CPU and Memory**
+
+   Enter the amount of virtual CPUs for the VM to use. Enter the amount
+   of memory, in MiB,for the VM to use. Not using adequate memory or
```

Comment:
The ", in MiB," is an aside.  We try to avoid these, but it's not necessary in this case anyway:
`` Enter the amount of memory in MiB for the VM. `` ("to use" at the end is not needed)

For the next sentence, the user is not "using" memory, they are giving it to the VM.  So:
`` Not providing adequate memory... ``